src/server/project.ts

```diff
@@ -1827,11 +1831,16 @@ namespace ts.server {
             super.close();
         }
 
+        setTypeAcquisition(newTypeAcquisition: TypeAcquisition): void {

```

Comment:
Wonder if if this.typeAcquisition  and set method should be moved to `Project` since 3 project types use the same method.